docs-content/index.html.md.erb

```diff
@@ -2,25 +2,15 @@
 title: a9s RabbitMQ for PCF
 owner: Partners
 ---
-<style>
-    .note.warning {
-        background-color: #fdd;
-        border-color: #fbb
-    }
-    .note.warning:before {
-        color: #f99;
-     }
-     </style>
-
-
- This documentation describes the a9s RabbitMQ for Pivotal Cloud Foundry (PCF) tile. a9s RabbitMQ for PCF
- enables on-demand provisioning of VM-based, dedicated RabbitMQ servers and clusters. Developers can
- create instances of a RabbitMQ server or cluster using Apps Manager or the Cloud Foundry Command Line
- interface (cf CLI) and bind these instances to an app.
- Depending on your service plan, a service instance may be associated with a single, dedicated VM
- or a set of VMs consisting of multiple VMs containing a RabbitMQ cluster.
-
-<%= image_tag("a9s-rabbitmq-tile.png") %>
+
+This documentation describes the a9s RabbitMQ for Pivotal Cloud Foundry (PCF) tile. a9s RabbitMQ for PCF
+enables on-demand provisioning of VM-based, dedicated RabbitMQ servers and clusters. Developers can
```

Comment:
Don't need the comma here. Generally sections separated by commas should be clauses i.e. would be valid sentences in themselves. Commas are also good for lists of 3 or more items. I prefer Oxford commas, where a comma comes after each item even the penultimate item before "and" or "or", however this was not what I was taught in school. It was considered correct, at least when I was at school, to omit the comma after the penultimate item.